sdk/python/kfp/dsl/databricks/README.md

```diff
@@ -0,0 +1,92 @@
+# Introduction to Databricks DSL 
+
+Databricks DSL provides a set of [Kubeflow Pipeline](https://www.kubeflow.org/docs/pipelines/) Tasks
+(Ops) which let us manipulate [Azure Databricks](https://azure.microsoft.com/services/databricks/) 
+resources using the [Azure Databricks Operator for Kubernetes](
+https://github.com/microsoft/azure-databricks-operator).
+
+These are the supported Ops up to date:
+
+- CreateClusterOp, to create a cluster in Databricks.
+- DeleteClusterOp, to delete an existing cluster from Databricks.
+- CreateJobOp, to create a Spark Job in Databricks.
+- DeleteJobOp, to delete an existing Spark Job from Databricks.
+- SubmitRunOp, to submit a one-time Run in Databricks.
+- DeleteRunOp, to delete an existing Run.
+
+We can create the Ops by using the complete Databricks spec as a Python dictionary or its individual
+parts as named parameters (not supported by Job Ops yet). SubmitRunOp also 
```

Comment:
Will this be supported before we push the real PR?

Reply:
Yes, that is why I changed the title of this PR to add [WIP]. I'm currently working on supporting named params for Job Ops too.

Reply:
Named params are supported in Job Ops now.